Monika After Story/game/script-topics.rpy

```diff
@@ -4666,6 +4666,24 @@ label monika_orchestra:
                  m 5a "Maybe we could do a duet someday!"
                  m 1j "Ehehe~"
                  $ persistent.instrument = True
+            elif tempinstrument == "harmonika":
+                 m 1hub "Wow, I've always wanted to try the harmonik--"
+                 m 3eub "...Oh!"
+                 m 1eka "Aww [player], did you do that for me?"
+                 m "That's {i}sooo{/i} adorable!"
+                 show monika 5eubfu at t11 zorder MAS_MONIKA_Z with dissolve
+                 m 5eubfu "And just so you know, you can play with me anytime you like..."
```

Comment:
🚂 

okay this definitely needs to be adjusted for affection.
Can you make variants of this bit of dialogue for varying affection levels?

Mainly add a non-racy reaction for affection levels below affectionate.